### PR TITLE
feat: respect my_clients_only param in sendh callback

### PR DESCRIPTION
**Pull request by PatrykBuniX**

### CODE REVIEW

**Review by atomrc — APPROVED** [review]

File: src/script/calling/CallingRepository.ts [hunk 1]
```diff
@@ -1129,6 +1129,9 @@ export class CallingRepository {
     targets: string | null,
     _unused: string | null,
     payload: string,
+    _len: number,
+    _trans: number,
+    my_clients_only: number,
```
Comment (atomrc) [inline src/script/calling/CallingRepository.ts]:
```suggestion
    myClientsOnly: number,
```

Why not?

File: src/script/conversation/MessageRepository.ts [hunk 2]
```diff
@@ -107,6 +107,7 @@ export interface MessageSendingOptions {
   /** Send native push notification for message. Default is `true`. */
   nativePush?: boolean;
   recipients?: QualifiedId[] | QualifiedUserClients | UserClients;
+  myClientsOnly?: boolean;
```
Comment (atomrc) [inline src/script/conversation/MessageRepository.ts]: I don't think this belongs here, the `MessageRepository` doesn't really care about this value

Comment (atomrc) [reply src/script/conversation/MessageRepository.ts]: I think instead it should only be an extra param of the `sendCallingMessage` of the calling repo

Comment (PatrykBuniX) [reply src/script/conversation/MessageRepository.ts]: 👍

**Review by atomrc — APPROVED** [review]

**Pull request merged**